otter/integration/tests/test_convergence.py

```diff
@@ -125,6 +125,13 @@ def oob_delete_then(self, rcs, scaling_group, num):
         Return a decorator that wraps a function call with logic to out-of-band
```

Comment:
Documentation: It might be more clear if this is stated to return a function that takes a function, rather than calling it a decorator.